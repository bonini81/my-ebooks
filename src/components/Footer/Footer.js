import React, { Component } from 'react';



const Footer = () => {
    return ( 

<React.Fragment>
        <footer className="container-fluid piepag-adb">

  <div className="row">
    
    <div className="col-12 col-md">
     <p className="piepag-parrafo">&copy;2020 Bonini81 - MyEbooks.</p>
    
    </div>
      
  </div>
</footer>

</React.Fragment>

     );
}
 
export default Footer;