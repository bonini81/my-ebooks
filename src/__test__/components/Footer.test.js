import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer/Footer';


describe('Footer Snapshot', () => {

test('Comprobar la UI del componente Footer sea siempre la misma Madafaka', () => {
    
    const footer = create(<Footer />);
    expect (footer.toJSON()).toMatchSnapshot();

});


});