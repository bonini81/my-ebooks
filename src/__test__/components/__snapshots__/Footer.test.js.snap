// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Comprobar la UI del componente Footer sea siempre la misma Madafaka 1`] = `
<footer
  className="container-fluid piepag-adb"
>
  <div
    className="row"
  >
    <div
      className="col-12 col-md"
    >
      <p
        className="piepag-parrafo"
      >
        Â©2020 Bonini81 - MyEbooks.
      </p>
    </div>
  </div>
</footer>
`;
